sdk/anomalydetector/ai-anomaly-detector-rest/src/anomalyDetectorRest.ts

```diff
@@ -7,17 +7,17 @@ import { AnomalyDetectorRestClient } from "./clientDefinitions";
 
 /**
  * Initialize a new instance of the class AnomalyDetectorRestClient class.
- * @param Endpoint type: string Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).
- * @param ApiVersion type: string Anomaly Detector API version (for example, v1.1).
+ * @param Endpoint type: string
+ * @param ApiVersion type: "v1.1"
  * @param credentials type: KeyCredential
  */
 export default function createClient(
   Endpoint: string,
+  ApiVersion: "v1.1",
```

Comment:
See https://github.com/joheredi/autorest.typescript/pull/1/files#r1022105262 for reference

Reply:
@joheredi I know we'll improve this feature in future, but I still have concern for the current implementation about extensible enum.  I init a pr to discuss about it - https://github.com/Azure/autorest.typescript/pull/1664. 

Could you take a look?